services/core-api/app/api/now_applications/resources/now_application_status_resource.py

```diff
@@ -132,16 +132,20 @@ def put(self, application_guid):
                             )
 
                     if contact.mine_party_appt_type_code != 'PMT' or new_permittee == True:
-                        mine_party_appointment = MinePartyAppointment.create(
-                            mine=now_application_identity.mine
-                            if contact.mine_party_appt_type_code != 'PMT' else None,
-                            permit=permit if contact.mine_party_appt_type_code == 'PMT' else None,
-                            party_guid=contact.party_guid,
-                            mine_party_appt_type_code=contact.mine_party_appt_type_code,
-                            start_date=datetime.utcnow(),
-                            end_date=None,
-                            processed_by=self.get_user_info())
-                        mine_party_appointment.save()
+                        current_mpa = permit.permittee_appointments[0]
```

Comment:
This will throw an index out of range error if permittee_appointments is empty

Reply:
this change is now irrelevant, I have reworked the whole flow to encompass MMG and PMT.